pkg/worker/worker.go

```diff
@@ -0,0 +1,244 @@
+/*
+Package worker allows to start, stop and query Linux processes as concurrent jobs.
+
+Each job is saved to a provided storage via DataStorage interface. Its progress
+is updated each time a new line in StdIn or StdErr appears.
+
+To make use of the package create a new Worker using its constructor - NewWorker
+and provide a correct DataStorage. Control the processes using Start() and Stop()
+methods. Query the output anytime, even when the process is running using Query()
+method.
+
+DataStorage interface must include working data synchronization methods: Lock()
+and Unlock().
+*/
+package worker
+
+import (
+	"bufio"
+	"log"
+	"os"
+	"os/exec"
+	"sync"
+	"syscall"
+	"time"
+
+	"github.com/pkg/errors"
+
+	googleUUID "github.com/google/uuid"
```

Comment:
As long as Stop and Query don't use any functions from the `uuid` package, it's safe to name the var `uuid`.
In case they need to call something else from package `uuid`, it's usually better to rename the var, to limit the scope of rename (package import name is file-wide, vs var which is function-wide)

Reply:
Makes sense, thanks for the explanation.